redis/src/connection.rs

```diff
@@ -201,6 +201,13 @@ impl IntoConnectionInfo for ConnectionInfo {
     }
 }
 
+/// URL format: `{redis|rediss}://[<username>][:<password>@]<hostname>[:port][/<db>]`
+///
+/// - Basic: `redis://127.0.0.1:6379`
+/// - Username & Password: `redis://user:password@127.0.0.1:6379`
+/// - Specifying DB: `redis://127.0.0.1:6379/database`
```

Comment:
Since database names are integers in Redis, can we change `database` to a `0` here?

```suggestion
/// - Specifying DB: `redis://127.0.0.1:6379/0`
```